wrapanapi/rhevm.py

```diff
@@ -109,9 +110,8 @@ def __init__(self, hostname, username, password, **kwargs):
             'url': url,
             'username': username,
             'password': password,
-            'filter': True,
-            'insecure': True
-        }
+            'insecure': True,
+            'filter': False if rhv_gt_4 else True}
```

Comment:
`not rhv_gt_4 `

Reply:
I disagree? With the existing filter=True, the queries return nothing against rhv41 providers, and worked for rhvm3.6 provider.

Reply:
Offline discussion, switched the logic up for readability, functionally correct.